themes/default/layouts/case-studies/list.html

```diff
@@ -31,6 +31,25 @@ <h5 class="leading-none">
                     </article>
                 </li>
             {{ end }}
+
+            <li class="m-0 p-0">
```

Comment:
It's a bit odd to tack this onto the end like this since it's not a case study like the others, but on the other hand it's nice to be able to surface things like this here as well. Not necessary for this PR, but figuring out how to list these as blog posts, maybe in a section of their own below, would probably be nice. (cc @susanev.)